spanner/tests/unit/test_pool.py

```diff
@@ -875,7 +876,7 @@ def __init__(self, name):
         self._sessions = []
 
     def session(self):
-        return self._sessions.pop()
+        return self._sessions.pop(0)
```

Comment:
`pop()` without args returns last element. So within tests list of sessions become reversed while putting it into pool. With this we would had to check, if first object from pool equals to first object from original sessions list, which looks like FIFO (it's not, 'cause original list would have been reversed twice before returning from pool, but it's not very obvious while looking at a test code). So it's easier to reverse order in mock object, and make test code more simple for understanding

Reply:
Perhaps add a comment in the code mentioning this.

Reply:
Okay, pushed